pkg/topics/topics.go

```diff
@@ -4,10 +4,11 @@ import (
 	"strings"
 
 	v1 "github.com/xmtp/example-notification-server-go/pkg/proto/message_api/v1"
+	"go.uber.org/zap"
 )
 
 const V1_PREFIX = "/xmtp/0/"
-const V3_PREFIX = "/xmtp/1/"
+const V3_PREFIX = "/xmtp/mls/1/"
```

Comment:
I literally have no idea how this ever worked. 🤔  @neekolas I swear V3 group messages worked at one point?

Reply:
Maybe that was so long ago it was before the topics changed?